catalog/app/components/TakeAction/index.js

```diff
@@ -20,7 +21,9 @@ const Container = styled.div`
 function TakeAction() {
   return (
     <Container>
-      <RaisedButton href={makeSignInURL()} label="Sign Up" primary />
+      {!cfg.team &&
```

Comment:
I recommend  something like this as it's more readable:
```
{ cfg.team ? null : <RaisedButton ... }
```

if the statement gets much longer than it's better (for future PRs) to compute the fragment above the return value and just inline it:
```
const optionalButton = ...
return (
  { optionalButton }
);
```

Reply:
Also, per your comment, it is indeed preferable to keep `TakeAction` as a dumb component and pass in a boolean to determine whether to show or hide.